cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfOperationsCreator.java

```diff
@@ -10,6 +10,9 @@
 
 public class CfOperationsCreator {
 
+    private static final String CF_CONTROL_USER = "CF_CONTROL_USER";
+    private static final String CF_CONTROL_PASSWORD = "CF_CONTROL_PASSWORD";
```

Comment:
A docstring describing that these constants are the names of the corresponding environment variables would be helpful, since their names do not imply that.